protoc-gen-grpc-gateway/descriptor/registry.go

```diff
@@ -50,12 +50,20 @@ type Registry struct {
 	// allowRepeatedFieldsInBody permits repeated field in body field path of `google.api.http` annotation option
 	allowRepeatedFieldsInBody bool
 
+	// repeatedPathParamSeparator specifies how path parameter repeated fields are separated
+	repeatedPathParamSeparator repeatedFieldSeparator
+
 	// useJSONNamesForFields if true json tag name is used for generating fields in swagger definitions,
 	// otherwise the original proto name is used. It's helpful for synchronizing the swagger definition
 	// with grpc-gateway response, if it uses json tags for marshaling.
 	useJSONNamesForFields bool
 }
 
+type repeatedFieldSeparator struct {
+	name string
+	sep  string
```

Comment:
If this is always a single character, should we just make it a `rune`?

Reply:
You are quite right. I will change this.